src/amber/scripts/environment_loader.cr

```diff
@@ -41,7 +41,7 @@ str = String.build do |s|
   #s.puts %(@log.level = #{settings["log_level"]? || "::Logger::INFO"})
   s.puts %(@color = #{settings["color"]? == nil ? true : settings["color"]})
   s.puts %(@redis_url = "#{settings["redis_url"]? || "redis://localhost:6379"}")
-  s.puts %(@port = #{settings["port"]? || 3000})
+  s.puts %(@port = #{ENV["PORT"]? || settings["port"]? || 3000})
```

Comment:
This should be `(ENV["PORT"] ||= "3000").to_i`

You can just do this on each of the environment YAMLs

```yaml
secret_key_base: <%= SecureRandom.urlsafe_base64(32) %>
port:  <%= (ENV["PORT"] ||= "3000").to_i %>
name: <%= @name %>
log: "::Logger.new(STDOUT)"
log_level: "::Logger::INFO"
color: true
...
```

Reply:
In this file it shouldn't be `(ENV["PORT"] ||= "3000").to_i` Just in the environment files. Do you prefer having it in each of the env yaml files? That was my original opinion but started second guessing that yesterday after reading your comments on it. 

Would it maybe be better to let port be overridden by ENV["PORT"] regardless of what's in the env.yml?

Reply:
@elorest I would expect the environment variable to override env.yml personally

Reply:
@sam0x17 We'll it will be setup that way by default in the env.yml's but it gives you the option to change it to a different ENV VAR if you wanted. The default will still have PORT override though it will just be configured in development.yml, prod, test etc.

Reply:
Either way this will be fixed for you.